script/emergency-access

```diff
@@ -0,0 +1,115 @@
+#!/bin/bash
+
+set -e
+
+# Script to manage emergency access configuration in S3 for login.gutools
+
+function usage {
+  echo "Usage: $0 [options]"
+  echo "Options:"
+  echo "  enable [stage] [--profile PROFILE]   - Enable emergency access for a stage"
+  echo "  disable [stage] [--profile PROFILE]  - Disable emergency access for a stage"
```

Comment:
The square brackets here imply to me that `options` and `stage` are optional, whereas they are actually required. I wonder about the following?

```suggestion
  echo "Usage: $0 <options>"
  echo "Options:"
  echo "  enable <stage> [--profile PROFILE]   - Enable emergency access for a stage"
  echo "  disable <stage> [--profile PROFILE]  - Disable emergency access for a stage"
```